06.bowling_object/frame.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require './shot'
+
+class Frame
+  attr_reader :shots, :idx
+
+  def initialize(shots, idx)
```

Comment:
なんとなくですが、「n番目のフレームのshots」と読めた方が自然な気がするのでidxが先に来た方がややベターかも。

```suggestion
  def initialize(idx, shots)
```